ci-cd/build-maven-project/action.yml

```diff
@@ -260,6 +260,23 @@ runs:
         unset-extra-envs "general" "${EXTRA_ENVS}"
         unset-extra-envs "from github" "${EXTRA_ENVS_FROM_GH}"
 
+    - name: Publish Test Results
+      uses: EnricoMi/publish-unit-test-result-action@567cc7f8dcea3eba5da355f6ebc95663310d8a07
+      if: always()
```

Comment:
kanskje `if: success() || failure()` her også?
Bare for å unngå trøbbel. Evt ser jeg `if: ${{ !cancelled() }}` foreslåes
ref. https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always